### ISSUE TITLE
5.6.10 won't let me publish posts

### ISSUE THREAD

**Issue by Beee4life** [elliotcondon/acf#672]
Seeing the update i immediately installed it, but when I did I noticed I couldn't publish any posts anymore. I tried pages and a custom post type. Updating did work. 

I reverted back to 5.6.9 and the issue is now gone.

**Comment by elliotcondon** [comment]
Hi @Beee4life 

Thanks for the bug report. This sounds very serious. Can you please provide some more thorough information about the issue?

**Comment by elliotcondon** [comment]
I've just setup a fresh WP site, and performed some testing but can't replicate this issue.
Can you please try to replicate the problem again and take note of your console log for any JS errors during the publish action.

**Comment by Beee4life** [comment]
I thought it did, so I reported it asap... I didn't dig further into it yet, because I quickly downgraded because I was 'on a roll' last night :) Didn't wanna dig deeper at that point... will look at it today...

**Comment by Beee4life** [comment]
Hmm that is weird, (luckily) I can't reproduce it anymore either... 

Last night it bugged me so hard that I downgraded immediately. The site I was working on is close to launch (after building on it for over 14 months) so I doubt there is some unexpected (custom) code that would f*ck it up.

And I know for a fact I didn't do anything weird since I was working on css and caching only. The only thing I can think of now, is that W3TC caused something unexpected.

**Comment by Beee4life** [comment]
I will continue to monitor.

**Comment by Beee4life** [comment]
I just installed a fresh new site. It's a default WP theme with only ACF Pro installed.

I got this error just now (when I tried to add a field to a group)

    Uncaught TypeError: Cannot read property 'field_object' of undefined
    http://phun.beee/content/plugins/advanced-custom-fields-pro/pro/assets/js/acf-pro-field-group.min.js?ver=5.7.0-beta1 line 1

I then checked if I could publish a page, which I can't.

**Comment by elliotcondon** [comment]
Hi @Beee4life 

Your JS error is coming from the file "acf-pro-field-group.min.js?ver=5.7.0-beta1".
- This is a BETA version, not "5.6.10" as you have previously stated.

Can you please confirm which version this issue is occurring one.
- Is it "5.6.10" or is it the beta "5.7.0-beta1"
- These are 2 very different versions

**Comment by Beee4life** [comment]
Damn. Was I drunk or so. Will check properly tomorrow.

**Comment by Beee4life** [comment]
You were right... 

It was caused by a grunt command which didn't clean all files properly, hence why it was left there... Sorry 'bout the confusion. Working on too many sites late at night.

I'll leave it/this topic for now...

I did notice something else, not a bug but imo a bit of unexpected UX. If you add a new field in a group. The title shows "New field". If you click the title, the title disappears and won't return.
